test_unfurl_message.py

```diff
@@ -185,3 +188,170 @@ def test_prepare_description__custom_width():
 )
 def test_get_handler(path_info, handler):
     assert uut.get_handler(path_info) == handler
+
+
+
+@pytest.mark.parametrize(
+    ("name", "api_resp", "want", "want_error"),
```

Comment:
The name is unused as far as I can tell. I assume you only wanted it there for nicer test runner output. In this case, you can set a keyword argument to `parametrize` called `ids` with the list of names instead, and then the test function will take one fewer arguments :relieved:

See here: https://docs.pytest.org/en/latest/example/parametrize.html#different-options-for-test-ids